src/GovUk.Education.ExploreEducationStatistics.Content.Api/Program.cs

```diff
@@ -21,10 +22,17 @@ public static void Main(string[] args)
                     webBuilder.UseStartup<Startup>();
                     webBuilder.ConfigureKestrel(serverOptions => { serverOptions.AddServerHeader = false; });
                 })
+                .ConfigureAppConfiguration((_, builder) =>
```

Comment:
Do we actually use FileTypeService in the Content API?  I thought Admin and Importer, but not Content potentially.

Reply:
Or is this more just for consistency if we want to support the use of Local settings in other projects down the line?

Reply:
This is just more for consistency so that we have local settings available in the future across the projects.

Reply:
Cool sounds good